js/packages/web/src/views/art/index.tsx

```diff
@@ -4,27 +4,77 @@ import {
   useConnection,
   useMeta,
   loadMultipleAccounts,
+  ENDPOINTS,
+  useLocalStorageState,
 } from '@oyster/common';
 import { useWallet } from '@solana/wallet-adapter-react';
-import { Button, List, Skeleton, Tag } from 'antd';
+import { Button, List, Skeleton, Tag, Space } from 'antd';
 import React, { useEffect, useState } from 'react';
+import { PublicKey } from '@solana/web3.js';
 import { useParams } from 'react-router-dom';
 import { sendSignMetadata } from '../../actions/sendSignMetadata';
 import { ArtContent } from '../../components/ArtContent';
 import { ArtMinting } from '../../components/ArtMinting';
 import { ViewOn } from '../../components/ViewOn';
 import { useArt, useExtendedArt } from '../../hooks';
 import { ArtType } from '../../types';
+import { holaSignMetadata } from '../../components/MintModal/sign-meta';
+
+const META_PROGRAM_ID = 'metaqbxxUerdq28cj1RbAWkYQm3ybzjb6a8bt518x1s';
```

Comment:
I think we have a list of all program ids we use in metaplex, maybe we should find that and add it there? @kespinola do you know where that list is?